MAINTAINERS

```diff
@@ -6,5 +6,6 @@ Jonathan Boulle <jonathan.boulle@coreos.com> (@jonboulle)
 Krzesimir Nowak <krzesimir.nowak@coreos.com> (@krnowak)
 Sergiusz Urbaniak <sergiusz.urbaniak@coreos.com> (@s-urbaniak)
 Stefan Junker <stefan.junker@coreos.com> (@steveeJ)
+Tamer Tas <tamer.tas@coreos.com> (@tmrts)
```

Comment:
@jonboulle I believe this change will increase the technical debt of rkt, please find a better solution